Helper/Data.php

```diff
@@ -72,19 +68,18 @@ public function log($value)
 
     public function createActivity($storeId, $type)
     {
-        $key = $this->getApiToken($storeId);
-        $secret = $this->getApiSecret($storeId);
+        $key      = $this->getApiToken($storeId);
+        $secret   = $this->getApiSecret($storeId);
+        $endPoint = $this->getApiEndpoint();
```

Comment:
The new tracking doesn't know anything about activities at the moment. This should be going to `p.metrilo.com`

Reply:
Fixed.